### ISSUE TITLE
Should "video element visual content has accessible alternative" (c5a4ea) maps to 1.2.8?

### ISSUE THREAD

**Issue by Jym77** [act-rules/act-rules.github.io#1391]
[video element visual content has transcript](https://act-rules.github.io/rules/1a02b0) is an atomic rule mapping to 1.2.8.
[video element visual content has accessible alternative](https://act-rules.github.io/rules/c5a4ea) is a composite rule using and **not** mapping to 1.2.8.

From a logical point of view, this makes no sense. If the composite rule is failing, then all of its atomic rules also are failing, and then 1.2.8 is not satisfied.
So, either the composite should also map to 1.2.8, or the atomic should not map to it.

I am not sure which one is the correct fix (on audio/video, both the SCs and our rules are somewhat hard to follow…)

@ShadowBB @WilcoFiers you wrote these rules. What do you think?

**Comment by WilcoFiers** [comment]
Jup, good point. Let me go and fix that.
